populate_db.py

```diff
@@ -120,6 +120,11 @@ def create_event_types():
         get_or_create(EventType, name=type_)
 
 
+def create_event_locations():
+    event_location = ['India', 'Singapore', 'Berlin', 'New York', 'Hong Kong']
+    for loc_ in event_location:
+        get_or_create(EventLocation, name=loc_)
+
```

Comment:
`create_event_locations` is added by you already :smile:

Reply:
AHHHHHHHHHHH

Reply:
@bhaveshAn Thank you, I don't know how that got there.